.travis.yml

```diff
@@ -23,11 +25,18 @@ matrix:
       env: SYMFONY_VERSION="2.7.*"
     - php: 5.6
       env: SYMFONY_VERSION="2.8.*" DEPENDENCIES=dev
+    - php: 7.1
+      env: SYMFONY_VERSION="2.8.*" DEPENDENCIES=dev
+    - php: 7.2
+      env: SYMFONY_VERSION="2.8.*" DEPENDENCIES=dev
     # Test against dev versions
     - php: 5.6
       env: DEPENDENCIES=dev
+    - php: 7.1
+      env: DEPENDENCIES=dev
```

Comment:
There is no reason to have 2 different jobs running `DEPENDENCIES=dev` tests here. Travis has a quota on concurrent jobs for the whole organization, so making the build matrix explode is not a good idea.